README.md

```diff
@@ -115,6 +104,8 @@ The Operations Team takes care of legal documents, invoicing and remittances.
 
 ## :pencil: Process
 
+> **:loudspeaker:** We ask teams to follow the process below, which has **faster processing times** and provides grants of up to **$30,000 for initial applications** and **$100,000 for follow-up** ones disbursed in Bitcoin, USDT or DAI. However, if you want to apply in **private**, need to be paid out in **fiat** and/or you **seek funding of $100,000+** and your work cannot be split into smaller milestones, you can apply [:arrow_right: here](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform).
```

Comment:
```suggestion
> **:loudspeaker:** Applications submitted through the process below are **public** and provide grants of up to **$30,000 for first-time applications** and **$100,000 for follow-up** ones. Payment is made in **Bitcoin, USDT or DAI**. If you want to apply in **private**, need to be paid out in **fiat** and/or you **seek funding above the aforementioned limits** and your work cannot be split into smaller milestones, you can apply [:arrow_right: here](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform).
```

Reply:
I think it's self-explanatory that they're public. If they're looking to make a private submission, it's in the next sentence.

Reply:
We'd also generally like to discourage people from using the form as much as possible, hence the "strongly encourage". But feel free to suggest alternatives.

Reply:
Commenting here so it's all in one thread

```suggestion
> **:loudspeaker:** We strongly encourage teams to follow the process below, which provides grants of up to **$30,000 for first-time applications** and **$100,000 for follow-up ones**. Payment is made in Bitcoin, USDT or DAI. If you want to apply in **private**, need to be paid out in **fiat** and/or you **seek funding above the aforementioned limits**, you can apply [:arrow_right: here](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform), but note that applications that follow the full process will be prioritised.
```

Reply:
> We'd also generally like to discourage people from using the form as much as possible

Why though? @Noc2?

Anyway, I'll let the owner of the PR decide. :)

Reply:
Open grants mean generally less work for us and are faster for grantees. But I would rather point out the benefits instead of the disadvantages. So I suggest removing “, but note that applications that follow the full process will be prioritised.” Also I don’t think we necessarily “prioritise” open grants.

Reply:
They're easier to manage: no form, no creating a contract, deliveries via PR, no committee/council approval. +everyone can see what's going on. Only advantages ;)

Reply:
If you are referring to my post, I meant the advantages for people that apply.

Reply:
@Noc2 it might make less sense to point out advantages since Open Grants are now just "normal grants"

Reply:
imo, "general" grants are there for a reason, and it shouldn't matter to the applicant if they mean more work for us -- if they have good reason to apply for one. Also, we currently have 2 `Editor accepted` general applications per month, so the overhead seems manageable.

Reply:
@semuelle something like the above?